### PR TITLE
tech: add snapshot test for add ons component

### PR DESCRIPTION
**Pull request by anusha5695**
## Related to issue: https://github.com/rahuldkjain/github-profile-readme-generator/issues/294

Snapshot test for addons component

### CODE REVIEW

**Comment by anusha5695** [comment]
@rahuldkjain  added snapshot test :)

**Comment by anusha5695** [comment]
@rahuldkjain could you review this please?

File: src/components/__tests__/addons.test.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import React from "react"
+import renderer from "react-test-renderer"
```
Comment (Citrinin) [inline src/components/__tests__/addons.test.js:2]: Thanks for your efforts, looks good!
But we shall rely only on one testing library use `enzyme-to-json` library and do not use `react-test-renderer`  - there is no trace of it in `master` now. Check how to use it in other test files.

Comment (anusha5695) [reply src/components/__tests__/addons.test.js]: Hi @Citrinin done

**Comment by anusha5695** [comment]
Hi @rahuldkjain , fixed merge conflicts and added 20 more tests to increase coverage as just snapshot test for the comment wasn't sufficient. Could you check plz?

**Review by zenby — APPROVED** [review]
Great @anusha5695 you've done a great job! 
LGTM! :+1:

**Comment by anusha5695** [comment]
Thank you @zenby ! 👍

**Review by Citrinin — APPROVED** [review]

**Pull request merged**